DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/storeFieldInIndex/metadata.json

```diff
@@ -0,0 +1,241 @@
+                    "Url": "/java/static-indexes/static-indexes-overview"
+                },
+                {
+                    "Title": "Map Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The index Entry class",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* The IndexEntry class defines the _index-fields per entry_ in the index.<br>These fields will be indexed and can be queried on.",
+                "",
+                "* The _content_ for each index-field is configured in the index constructor (see next step).",
+                "",
+                "* Field _totalItemsOrdered_ will be computed and stored in the index (Step 5)."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The projected class",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* `OrderProjectedDetails` will contain the details projected per document in the results.<br>",
+                "  Used in step 6.",
+                "",
+                "* Note: it is not mandatory for this class to be nested within the index class."
+            ],
+            "Assets": [
+                {
+                    /*TODO
+                    "Title": "Project Index Result Demo",
+                    "Type": "demo",
+                    "Url": "/java/static-indexes/project-index-results"
+                    */
+                }
```

Comment:
Need to put the opening and closing brackets `{` & `}`  also in a comment
other wise you get an empty link in UI

Reply:
Done